WordPress/Sniff.php

```diff
@@ -550,6 +550,76 @@ public function strip_quotes( $string ) {
 	}
 
 	/**
+	 * Add a PHPCS message to the output stack as either a warning or an error.
+	 *
+	 * @since 0.11.0
+	 *
+	 * @param string $message   The message.
+	 * @param int    $stackPtr  The position of the token the message relates to.
+	 * @param bool   $is_error  Optional. Whether to report the message as an 'error' or 'warning'.
+	 *                          Defaults to true (error).
+	 * @param string $code      Optional error code for the message. Defaults to 'Found'.
+	 * @param array  $data      Optional input for the data replacements.
+	 * @param int    $severity  Optional. Severity level. Defaults to 0 which will translate to
+	 *                          the PHPCS default severity level.
+	 * @return bool
+	 */
+	protected function addMessage( $message, $stackPtr, $is_error = true, $code = 'Found', $data = array(), $severity = 0 ) {
```

Comment:
Instead of `$is_error` being a boolean, should it perhaps be a string `error` or `warning`? Just thinking that in the future I would hope PHPCS would add an informational message that isn't error or warning. Also, passing `'error'` is more readable than passing `true`.